api/src/routers/meals.js

```diff
@@ -0,0 +1,56 @@
+import express from "express";
+import knex from "../database_client.js";
+
+const meals = express.Router();
+
+meals.get("/", async (req, res) => {
+  const meals = await knex("Meal").orderBy("ID", "asc");
+  res.send(meals);
+});
+
+meals.post("/", async (req, res) => {
+  const body = req.body;
+
+  if (body) {
+    await knex("Meal").insert(body);
+  }
+});
+
+meals.get("/:id", async (req, res) => {
+  const mealId = parseInt(req.params.id, 10);
```

Comment:
oh, interesting. can you explain this line?

Reply:
Oh, when I tried like `mealID  = req.params.id ` I wasnt able to get the parameter from the url. Thats why I have used I thought like maybe I needed to make it integer. But I have tried now as mealID = `req.params.id ` and it worked. So I probably got error of something else there :)